alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md

```diff
@@ -16,8 +17,9 @@ leading to data loss if not addressed promptly.
 ## Diagnosis
 
 The alert triggers when a Persistent Volume Claim (PVC) approaches or surpasses
-75% of its capacity. It indicates the need to expand the PVC size to accommodate
-more data or to remove unnecessary data to free up space.
+very high capacity limit. It indicates the need to expand the PVC size to
+accommodate more data or to remove unnecessary data to free up space.
+Please see the alert doc for exact threshold limit.
```

Comment:
Why not use the same text explanation in all the runbooks?
_"Please see the alert documentation text for an exact threshold limit."_

Reply:
Done